### ISSUE TITLE
Error when only one auth type is defined

### ISSUE THREAD

**Issue by wojtek-oledzki** [ansible-city/composer#3]
Actual:

```
roles:
  - role: ansible-city.composer
    composer:
      user: test_single_auth
      auth:
        github:
          github.com: qwerty654321

...
TASK [ansible-city.composer : Create composer github auth] *********************
fatal: [default]: FAILED! => {"changed": false, "failed": true, "msg": "ValueError: Circular reference detected"}
```

Expected: no errors

**Comment by wojtek-oledzki** [comment]
PR created: #4

**Comment by wojtek-oledzki** [comment]
merged and released in v1.0.3
